code/modules/cargo/bounties/chef.dm

```diff
@@ -226,17 +238,14 @@
 	reward_high = 5300
 	required_count = 3
 	random_count = 1
-	wanted_types = list(/obj/item/reagent_containers/food/snacks/bacon_stick,
-			/obj/item/reagent_containers/food/snacks/egg_pancake,
-			/obj/item/reagent_containers/food/snacks/hatchling_suprise,
-			/obj/item/reagent_containers/food/snacks/red_sun_special,
-			/obj/item/reagent_containers/food/snacks/riztizkzi_sea,
-			/obj/item/reagent_containers/food/snacks/sliceable/grilled_carp,
-			/obj/item/reagent_containers/food/snacks/sliceable/sushi_roll,
-			/obj/item/reagent_containers/food/snacks/stuffed_meatball,
-			/obj/item/reagent_containers/food/snacks/chilied_eggs,
-			/obj/item/reagent_containers/food/snacks/sliceable/eyebowl,
-			/obj/item/reagent_containers/food/snacks/hatchbowl)
+	wanted_types = list(/obj/item/reagent_containers/food/snacks)
+
+/datum/bounty/item/chef/unathi/applies_to(var/obj/item/reagent_containers/food/snacks/O)
+	if(!..())
+		return FALSE
+	if(O.icon == 'icons/obj/item/reagent_containers/food/cultural/unathi.dmi')
+		return TRUE
```

Comment:
checking the icon isn't an acceptable way to do this - icon files are prone to being changed and generally should never be used to check the state or type of an object